builtin/logical/pki/cert_util.go

```diff
@@ -958,7 +968,7 @@ func generateCreationBundle(b *backend, data *dataBundle) error {
 		KeyBits:                       data.role.KeyBits,
 		NotAfter:                      notAfter,
 		KeyUsage:                      x509.KeyUsage(parseKeyUsages(data.role.KeyUsage)),
-		ExtKeyUsage:                   extUsage,
+		ExtKeyUsage:                   extUsage | parseExtKeyUsages(data.role.ExtKeyUsage),
```

Comment:
If you send `data.role` into parseExtKeyUsages then you can move the logic up above into it so this just becomes `ExtKeyUsage: parseExtKeyUsages(data.role)`.